src/main/webapp/app/course/plagiarism-cases/student-view/detail-view/plagiarism-case-student-detail-view.component.ts

```diff
@@ -48,6 +59,23 @@ export class PlagiarismCaseStudentDetailViewComponent implements OnInit, OnDestr
         });
     }
 
+    /**
+     * Get the url segment for different types of exercises.
+     * @param exerciseType type of exercise
+     */
+    getExerciseUrlSegment(exerciseType?: ExerciseType) {
```

Comment:
This is an exact duplicate of the method in ``plagiarism-case-instructor-detail-view.component.ts``. Any way we could avoid introducing this duplication, especially in the same domain (plagiarism)?